### PR TITLE
Bump 12.2.2

### PR DESCRIPTION
**Pull request by juanedi**
Patch to fix a bug in our Modal's close button.

### CODE REVIEW

**Review by tesk9 — COMMENTED** [review]
@juanedi I think there's some accidentally-committed files in here -- this is meant to just be elm.json, right?

**Comment by juanedi** [comment]
ouchhhhh! fixing 😅

File: src/ModalExample.elm [hunk 1]
```diff
@@ -0,0 +1,116 @@
+module ModalExample exposing (..)
```
Comment (tesk9) [inline src/ModalExample.elm:1]: I don't think we want this file either, do we...?

Comment (juanedi) [reply src/ModalExample.elm]: not sure what's wrong! git kept re-adding that file. should be fixed now.

**Review by tesk9 — APPROVED** [review]
Woo!

**Comment by tesk9** [comment]
Rebased on master, so this will now do the following:

```
This is a MINOR change.

---- ADDED MODULES - MINOR ----

    Nri.Ui.SegmentedControl.V13
    Nri.Ui.Tabs.V7
    Nri.Ui.Tooltip.V2
```

**Comment by tesk9** [comment]
Oh wait but then that means this is a minor change rather than a patch

**Comment by tesk9** [comment]
Published!

**Pull request merged**